src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  return sourceString
+    .split(';')
+    .map((styles) => styles.trim())
+    .filter((styles) => styles !== '')
+    .reduce((cssStyles, styles) => {
+      const [key, value] = styles.split(':').map((item) => item.trim());
+
+      if (key && value) {
+        cssStyles[key] = value;
+      }
+
+      return cssStyles;
+    }, {});
 }
 
 module.exports = convertToObject;
```

Comment:
The `module.exports` is correctly used to export the `convertToObject` function, but make sure to test the function thoroughly, including edge cases where style declarations might contain colons or semicolons within the values.